frontend/src/utils/request-query-params/index.ts

```diff
@@ -0,0 +1,9 @@
+export const getQueryFormattedUrl = (queryObject: { [key: string]: string }) => {
+  const queryFormattedUrl: string[] = [];
+
+  Object.entries(queryObject).forEach(([key, value]) => {
+    queryFormattedUrl.push(`${key}=${value}`);
+  });
+
+  return queryFormattedUrl.join('&');
+};
```

Comment:
```suggestion
export const getQueryFormattedUrl = (queryObject: { [key: string]: string }) => {
  const queryFormattedUrl = Object.entries(queryObject).map(([key, value]) => `${key}=${value}`);

  return queryFormattedUrl.join('&');
};
```


직접 돌려본건 아닌데 이건 어떻습니까?

Reply:
오 굿굿 저게 훨씬 깔끔하네!